tests/desktop/consumer_pages/test_users_account.py

```diff
@@ -15,15 +15,11 @@
 
 class TestAccounts(BaseTest):
 
-    @pytest.mark.credentials
-    def test_create_new_user_using_API(self, mozwebqa):
-
+    def test_create_new_user_using_fxapom(self, mozwebqa, new_user):
```

Comment:
The need for this test is questionable as we are exercising this same set of steps in a number of other tests (where we are creating a new FxA account and then using it to log in to Marketplace).

@krupa can we delete this test?